plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AbstractDeltaLakePageSink.java

```diff
@@ -138,6 +138,7 @@ public AbstractDeltaLakePageSink(
         ImmutableList.Builder<DeltaLakeColumnHandle> dataColumnHandles = ImmutableList.builder();
         ImmutableList.Builder<Type> dataColumnTypes = ImmutableList.builder();
         ImmutableList.Builder<String> dataColumnNames = ImmutableList.builder();
+        ImmutableList.Builder<OptionalInt> dataColumnFieldIds = ImmutableList.builder();
```

Comment:
Do we need this assignment in this current context (focusing on `name` column mapping)?
Same question for the `ParquetSchemaConverter` changes.

Reply:
removed